docs/dev_guide/software-required.md

```diff
@@ -0,0 +1,55 @@
+# Get the required software for Linux and MacOS
+Here we will explain the software you need to use Linux or MacOS for `kind`
+development.
+Before you being you must have:
+
+* GitHub account
+* `git`
+* `go`
+* `Docker`
+
+## Task 1. Get a GitHub account
+To contirbute to `kind`, you will need a [GitHub][github] account.
+
+Make sure to read you read the [Kubernetes community guidelines][community].
+In specific, read through the [Kubernetes contributor guidelines][contributor].
+
+## Task 2. Install git
+Install `git` on your local machine.
+You can check if `git` is already on your system and properly installed with 
+the following command:
+
+```
+$ git --version
+```
+This documentation is written using `git` version 2.17.1. 
+Your version may be different depending on your OS.
+
+## Task 3. Install or upgrade Go
+Install or upgrade [Go using the instructions for your operating system][golang].
+You can check if Go is in your system with the following command:
+
+```
+$ go version
+```
+This documentation is written using Go version 1.10.4.
```

Comment:
`kind` uses 1.11+ currently, due to `go fmt` changes in 1.11

Reply:
:heavy_check_mark: